### ISSUE TITLE
Thumbnails generator for GLB/GLTF objects

### ISSUE THREAD

**Issue by zmnv** [hicetnunc2000/hicetnunc#746]
Hello. Does hicetnunc.xyz has thumbnail generation process when minting? Because all objects (i.e. the first 85k objects dataset) has the same thumbnail now:

<img src="https://cloudflare-ipfs.com/ipfs/QmNrhZHUaEqxhyLfqoq1mtHSipkWHeT31LNHb1QEbDHgnc" width="128" />

Actually the creation process is very simple but.. I don't know how to store files right way... 

Many object's sources has size around 40mb... it's very big when you try to see a lot of objects in a list.

PS: now I have something like that just to see hen objects catalog ONLY with 3d models with the same model viewer module.
https://www.youtube.com/watch?v=eqLrYNmI2xw

**Comment by andrevenancio** [comment]
There is already work underway to generate other metadata fields besides `artifactUri`. We're generating `displayUri` and `thumbnailUri` for all old OBJKT before enabling this feature. It takes a while.

Secondly, as personal preference from hicetnunc founder we're deciding to explicitely disable the thumbnailUri generation [here](https://github.com/hicetnunc2000/hicetnunc/blob/main/src/data/ipfs.js#L42)
@tarwin has been generating all the previous displayUri. I think he already has a solution for GLB files.

**Comment by tarwin** [comment]
Yeah, I've got a solution for every file time. I've generated up to OBJKT 60,000 so far and ready to do the rest.

**Comment by ertdfgcvb** [comment]
> We're generating `displayUri` and `thumbnailUri` for all old OBJKT before enabling this feature. 

Slightly unrelated question: will the thumbnails also be generated for (interactive) SVGs uploaded without .ZIP packaging?

**Comment by tarwin** [comment]
The solution I have does create for interactive SVGs. I don't know if the frontend one does.

**Comment by zmnv** [comment]
Im done with my solution. I use it only for 3d objects. Sooooooo now there is 'heo gallery' and anyone can find the link on hen discord channel #3rd-party-tools.

Soooo... I can actualize my question — will hen create 'poster' automatically or user should upload thumbnail image as now forever?

I mean something like this but automatically:
<a href="https://user-images.githubusercontent.com/25253501/121249492-51128e00-c8ad-11eb-9dc4-7b250cdf9c44.png"><img src="https://user-images.githubusercontent.com/25253501/121249492-51128e00-c8ad-11eb-9dc4-7b250cdf9c44.png" width="256"/>
https://modelviewer.dev/examples/tester.html

1. User upload one glb/gltf file
2. Make a poster (automatically OR manual by user input when preview or something).
3. Prepare object. Prepare thumbnail (generate poster and upload it with object source)
3. meta "thumbnailUri" to this poster file

**Comment by AlexandreRangel** [comment]
Or this SVG icon (document with code inside :)
[https://github.com/microsoft/vscode-codicons/blob/main/src/icons/file-code.svg](https://github.com/microsoft/vscode-codicons/blob/main/src/icons/file-code.svg)
